Azure/communication-ui-library-android
# [Feature] API 21 and 22 support

## Purpose

* This PR is to support Android API 21 and 22.  We were not supporting API 21 and 22 because of theming issue, and this PR fixes the issue.
* Emulator crashes for API 21 and 22 (on call screen) - known issue

## Impact
* call join button, call end buttons
* Speaker selection icons
* video, mic mute/unmute icon
* setup screen disabled video icon color
* participant audio/video/pip background color
* Video view info header background color

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code
Please make sure theming is working fine from API 21+ onwards. 
If you do not have device, confirm with emulator on API 23+
Verify with dark and light theme
